email-config-shell/shell-definition.yaml

```diff
@@ -0,0 +1,64 @@
+tosca_definitions_version: tosca_simple_yaml_1_0
+
+metadata:
+  template_name: Email Config Shell
+  template_author: gaston.g
+  template_version: 0.1.0
+  template_icon: shell-icon.png
+
+description: >
+  TOSCA based resource shell
+
+imports:
+  - cloudshell_standard: cloudshell_resource_standard_2_0_3.yaml
+
+node_types:
+
+  vendor.resource.Email Config Shell:
+    derived_from: cloudshell.nodes.GenericResource
+    properties:
+      SMTP Server:
+        type: string
+        description: SMTP server address
+      SMTP Port:
+        type: string
+        description: SMTP server port
+      From Address:
+        type: string
+        description: From email address
+      Portal URL:
+        type: string
+        description: Portal URL
```

Comment:
I think that the Portal URL is missing from the capabilities so it will not be displayed in the UI when creating the resource

Reply:
Added